submitter/s3.py

```diff
@@ -0,0 +1,29 @@
+import logging
+from typing import List
+
+import boto3
+
+logger = logging.getLogger(__name__)
+
+
+def check_s3_permissions(buckets: List[str]) -> str:
+    """Checks S3 ListObjectV2 and GetObject permissions for all buckets provided in the
+    passed list. If either command is not allowed for any of the provided buckets,
+    raises an Access Denied bocotore client error.
+    """
+    s3 = boto3.client("s3")
+    bucket_names = []
+    for bucket in buckets:
```

Comment:
This isn't a super real world production/staging concern, but I'm curious if we'd run into issues test infra with empty s3 buckets here. This isn't blocking but do we want to report if the s3 bucket is empty here in addition to whether it could list stuff? That seems maybe useful but maybe not.

Reply:
Huh. Yeah I guess if the bucket is empty we can't really check the get object permission. But that would require someone (most likely Aaron) to go put a file in the bucket and run again to confirm. @zotoMIT thoughts?

Reply:
i think the way its written is fine.  I can always place a file by hand if I need, or via terraform like i do with the ssm  "secure" parameter.